Blockzilla/URLBar.swift

```diff
@@ -291,10 +291,10 @@ class URLBar: UIView {
             if inBrowsingMode {
                 make.trailing.equalTo(safeAreaLayoutGuide)
             }else {
-                make.trailing.equalTo(safeAreaLayoutGuide).offset(-UIConstants.layout.urlBarMargin)
+                make.trailing.equalTo(safeAreaLayoutGuide).offset(-14)
```

Comment:
You should add a new variable to UIConstants.layout with the value 14, we should not place values explicitly